Makefile.common

```diff
@@ -174,6 +174,15 @@ common-test: $(GOTEST_DIR)
 $(GOTEST_DIR):
 	@mkdir -p $@
 
+.PHONY: save-tests-results
```

Comment:
This needs to go in Makefile first if it's not going into prometheus/Makefile.common first.